packages/phpstan-latte-rules/src/Contract/LatteTemplateHolderInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanLatteRules\Contract;
+
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPStan\Analyser\Scope;
+use Symplify\LattePHPStanCompiler\ValueObject\ComponentNameAndType;
+use Symplify\TemplatePHPStanCompiler\ValueObject\RenderTemplateWithParameters;
+
+interface LatteTemplateHolderInterface
```

Comment:
Working name for now. I want to cover all classes which somehow "hold" / "work" with template

Reply:
These services should have tests, so we can fix them easily when broken.

Reply:
they are tested in LatteCompleteCheckRuleTest.php (original tests + presenter tests)

Reply:
I know, I meant separate test cases desired for this logic, as LatteCompleteCheckRuleTest covers everything.
It's hard to see what exactly breaks, as this rule is extreemely complex compared to normal PHPStan rules we have.

Reply:
ok I'll add some tests